content/en/docs/contribute/generate-ref-docs/metrics-reference.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Generating reference documentation for metrics
+content_type: task
+weight: 100
+---
+
+<!-- overview -->
+
+This page demonstrates the generation of metrics reference documentation
+
+## {{% heading "prerequisites" %}}
+
+{{< include "prerequisites-ref-docs.md" >}}
+
+<!-- steps -->
+
+## Cloning the Kubernetes repository
+
+The metric generation happens in the Kubernetes repository. 
+To clone the repository, Change directories to where you want the clone to exist.
+
+Then, execute the following command:
+
+```shell
+git clone https://www.github.com/kubernetes/kubernetes 
+```
+
+This creates a `kubernetes` folder in your current working directory.
+
+## Generate the metrics
+
+Inside the cloned repository, You will find a `test/instrumentation/documentation` directory.
+This is where the metrics are generated and maintained
+
+With each release, new metrics are added and the generated documentation must be kept up to date.
+
+To generate the latest metrics, Make sure you are in the root of the cloned Kubernetes directory. 
+Then, execute the following command:
+
+```shell
+./test/instrumentation/update-documentation.sh
+```
+
+To check for changes, execute:
+
+```shell
+git status
+```
+
+The output is similar to
+
+```
+./test/instrumentation/documentation/documentation.md
+./test/instrumentation/documentation/documentation-list.yaml
+```
+
+## Copy the generated metrics over to the website repository
+
+### Setting the website root
+
+execute the following command to set the website root:
+
+```shell
+export WEBSITE_ROOT=<path to website root>
+```
+
+replace `<path to website root>` with the path your kubernetes website root repository resides in
+
+### Copy operation
+
+This should copy the generated metrics over to the website repository
+
+```shell
+cp ./test/instrumentation/documentation/documentation.md "${WEBSITE_ROOT}/content/en/docs/reference/instrumentation/metrics.md"
+```
+
+{{< note >}}
+If you get an error about privileges, check that you haven't set the ownership incorrectly.
+You can use `chown` to change the file ownership back to your own user.
+{{< /note >}}
+
+## Creating a pull request
+
+To create a pull request for the website, Follow these steps:

```

Comment:
This information is already found in the contributing pages. Can you link to that section instead of repeating the commands on this page?